lib/issues.go

```diff
@@ -47,10 +43,11 @@ func CompareIssues(config cfg.Config, ghClient clients.GitHubClient, jiraClient
 	for _, ghIssue := range ghIssues {
 		found := false
 		for _, jIssue := range jiraIssues {
-			id, _ := jIssue.Fields.Unknowns.Int(config.GetFieldKey(cfg.GitHubID))
-			if int64(*ghIssue.ID) == id {
+
+			id, _ := config.GetFieldMapper().GetFieldValue(&jIssue, cfg.GitHubID)
+			if int64(*ghIssue.ID) == id.(int64) {
 				found = true
-				if err := UpdateIssue(config, ghIssue, jIssue, ghClient, jiraClient); err != nil {
+				if err := UpdateIssue(config, ghIssue, jIssue, ghClient, jiraClient,); err != nil {
```

Comment:
Is this comma intentional?

Reply:
No. Another peculiarity of go syntax that this is allowed. I'll remove it.